pkgs/by-name/do/doubleratchet/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  python3Packages,
```

Comment:
We should remove `python3Packages` and explicitly call every dependency, here. Example:

```nix
{
  lib,
  buildPythonPackage,
  setuptools,
  ...
}:
```